ui/apps/platform/src/Containers/PolicyCategories/PolicyCategoriesList.tsx

```diff
@@ -1,27 +1,38 @@
 import React from 'react';
 import { SimpleList, SimpleListItem } from '@patternfly/react-core';
 
+import { PolicyCategory } from 'types/policy.proto';
```

Comment:
If you rebase from master, you can import `PolicyCategory` from services/policyCategoryService.ts instead.

Reply:
I'm working on this PR while Linda is out, and I'm confused by this. I thought we were moving our type definitions to the `types/<x>.proto.ts` files, where each file mapped to a backend API proto?

Reply:
The frontend file corresponds to backend file which declares the the type, so all of us know where to compare. A recent trend is for service to declare it response types.